data/human/human missions.txt

```diff
@@ -3584,3 +3585,42 @@ mission "Mafia Extortion"
 			label payup
 			`	The man takes your credits, responds with a brief "Thank you," and walks away.`
 				accept
+
+
+
+mission "Courier 5"
+	name "Yottrite samples to <planet>"
+	deadline
+	description "Deliver a few Yottrite samples to <destination> by <date>. Payment is <payment>."
+	minor
+	cargo "Yottrite samples" 10
+	to offer
+		random < 5
+	source
+		government "Quarg"
+	destination
+		distance 2 13
+		government "Republic"
+		attributes "rich"
+	on offer
+		conversation
+			`A man waves at you from afar, trying to get your attention."`
+			choice
+				`	(Find out what he wants.)`
+				`	(Ignore him.)`
+					decline
+			`	"My name is Jason ," he says. "I've gotten my hands on a few Yottrite samples that I bought from a Quarg. We need you to send them to a research center."`
+			choice
+				`	"I'd be glad to."`
+				`	"Sorry, I'm not interested in that sort of work."`
+					decline
+			`	"Excellent!" he says. "Come, let my men load it up onto your ship. An associate of mine will pick them up at <destination>." He adds, "They are somewhat unstable, so be careful with them or they could explode."`
+				accept
+	
+	on visit
+		dialog phrase "generic cargo on visit"
+	on complete
+		log "Minor People" "Jason" `A wealthy individual who appears to be involved in researching Yottrite.`
```

Comment:
I don't think we get enough character development for Jason to justify a log entry for him as-is. I'd suggest fleshing out his character a bit more.